src/App.js

```diff
@@ -1,23 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { useTranslation } from "react-i18next";
 
 function App() {
+  const { t } = useTranslation();
```

Comment:
t kya h samjh nhi arha h,koi usefull name de do

Reply:
{ t } is general convention for using translation, refer docs